images/ansible-operator/Dockerfile

```diff
@@ -17,12 +17,13 @@ RUN yum clean all && rm -rf /var/cache/yum/* \
   && yum -y update \
   && yum install -y libffi-devel openssl-devel python36-devel gcc python3-pip python3-setuptools \
   && pip3 install --no-cache-dir \
-    ipaddress \
     ansible-runner==1.3.4 \
     ansible-runner-http==1.0.0 \
-    openshift~=0.10.0 \
-    ansible~=2.9 \
-    jmespath \
+    ipaddress==1.0.23 \
+    kubernetes==10.1.0 \
+    openshift==0.10.3 \
+    ansible==2.9.15 \
+    jmespath==0.10.0 \
```

Comment:
Question for a follow-up: can we pin these to major/minor versions and permit patch upgrades with [`~=`](https://www.python.org/dev/peps/pep-0440/#compatible-release)?

Reply:
Since this is the base image, I'm convinced these should be pinned for reproducibility.